File: FromSlides/Fortran/10_single_thread.f90
PROGRAM single_thread

  USE omp_lib

 !$OMP PARALLEL
    PRINT *,'Parallel from thread ', omp_get_thread_num()
 !$OMP SINGLE
    PRINT *,'Single from thread ', omp_get_thread_num()
 !$OMP END SINGLE
 !$OMP END PARALLEL

END PROGRAM single_thread

File: FromSlides/C/10_single_thread.c
#include <stdio.h>
#include <omp.h>

int main(){


 #pragma omp parallel
   printf("Parallel from thread %i\n", omp_get_thread_num());
 #pragma omp single
  {
    printf("Single from thread %i\n", omp_get_thread_num());
  }
}
